recipes-kernel/linux/files/prplmesh.cfg

```diff
@@ -0,0 +1,28 @@
+CONFIG_TUN=y
+CONFIG_DUMMY=y
+CONFIG_BRIDGE=y
+CONFIG_BRIDGE_NETFILTER=y
```

Comment:
For completeness, we should probably also select NETFILTER and NF_TABLES.

Reply:
Done